man/dracut.conf.5.adoc

```diff
@@ -370,7 +370,8 @@ The initrdname configuration option is required to match the
 _initr*$\{kernel\}*_ file pattern and only one file with this pattern should
 exists in the directory where initramfs is loaded from.
 +
-Defaults to _initramfs-$\{kernel\}.img_.
+Defaults to _initramfs-$\{kernel\}.img_. Debian/Ubuntu based systems normally
+ship a configuration file to change the defaults to _initrd.img-$\{kernel\}_.
```

Comment:
Please drop this change.

Reply:
Make it more generic, since it's not just distributions, anyone can make a snippet to change the name format.

Reply:
I removed the man page change.